avaframe/com1DFA/com1DFA.py

```diff
@@ -251,7 +253,7 @@ def com1DFACoreTask(simDict, inputSimFiles, avalancheDir, outDir, cuSim):
     return simDF, tCPUDF, dem, reportDict
 
 
-def com1DFAPostprocess(simDF, tCPUDF, simDFExisting, cfgMain, dem, reportDictList, exportData):
+def com1DFAPostprocess(simDF, tCPUDF, simDFExisting, cfgMain, dem, reportDictList, exportData, adaptedDemBackground=False):
```

Comment:
This addition to the parameters is a very specialised topic vs the general nature of the function. Could this be deducted from / included in cfgs? Or, if the adapted DEM is wanted, supply it as the dem ?